src/Resource/__tests__/links.spec.ts

```diff
@@ -188,7 +192,39 @@ describe('Incoming / Outgoing Links behavior', () => {
         from: 0,
         size: 20,
       });
+      expect(resource.expanded).toEqual({
+        '@id': 'https://myExpandedResourceURl.com/someID',
+      });
+      expect(fetchMock.mock.calls[2][1].body).toMatchSnapshot();
+      expect(links.results[0]).toHaveProperty('predicate');
+      expect(links.results[0]).toHaveProperty('link');
+      expect(links.results[0].link).toBeInstanceOf(Resource);
+      fetchMock.resetMocks();
+    });
 
+    it('should work having already the expanded resource', async () => {
+      const resource = new Resource<{
+        subject: string;
+      }>('testOrg', 'testProject', mockGetByIDResourceResponse);
+      resource.expanded = {
+        '@id': 'https://myExpandedResourceURl.com/someID',
+      } as any;
```

Comment:
I guess we could type` ExpandedResource` as we now it have `@id`, `@type` and `[keys: string]: any`

Reply:
you got it buddy